components/webui/imports/api/search/server/publications.js

```diff
@@ -78,15 +90,20 @@ Meteor.publish(Meteor.settings.public.SearchResultsCollectionName, ({
  * @param {string} publicationName
  * @param {object} props
  * @param {string} props.aggregationJobId
+ * @param {boolean} props.isExpectingUpdates Whether the subscriber
+ * expects that the collection will be updated.
  * @return {Mongo.Cursor} cursor that provides access to the aggregation results.
  */
 Meteor.publish(Meteor.settings.public.AggregationResultsCollectionName, ({
     aggregationJobId,
+    isExpectingUpdates,
 }) => {
     const collection = searchJobCollectionsManager.getOrCreateCollection(aggregationJobId);
```

Comment:
As done for `SearchResultsCollectionName`, let's log all subscription arguments.